api/tests/integration/infrastructure/repositories/organization-invitation-repository_test.js

```diff
@@ -199,37 +199,68 @@ describe('Integration | Repository | OrganizationInvitationRepository', function
   });
 
   describe('#findPendingByOrganizationId', function () {
-    const organizationId = 6789;
+    it('should find two of the three organization-invitations from db by organizationId', async function () {
```

Comment:
suggestion `should find only pending organization-invitations from db by organizationId`

Reply:
ok ❤️